### ISSUE TITLE
Add WCAG 2.1 new contrast-related 1.4.11 Non-text contrast (AA)

### ISSUE THREAD

**Issue by patrickhlauke** [ThePacielloGroup/CCAe#20]
(Dumping from the skype conversation on this)

we'd also want to start thinking about showing the other contrast-related SC info in 2.1 - 1.4.11 non-text contrast (AA)

that makes it slightly less slick. think it's worth deemphasising the pass/fail a bit, concentrating on showing the contrast ratio first and foremost, and then having explicit (but smaller) note of 1.4.3 (AA), 1.4.3 (AA) Large Text, 1.4.6 (AAA), 1.4.6 (AAA) Large Text, 1.4.11 (AA) Non-text
